Tests/SwiftDocCTests/Rendering/RenderNodeTranslatorTests.swift

```diff
@@ -1225,6 +1225,9 @@ class RenderNodeTranslatorTests: XCTestCase {
             roundTrippedArticle.metadata.color?.standardColorIdentifier,
             "yellow"
         )
+
+        XCTAssertEqual(roundTrippedArticle.metadata.roleHeading, "Book-Like Content")
+        XCTAssertEqual(roundTrippedArticle.metadata.role, "article")
      }
     
     func testPageColorMetadataInSymbolExtension() throws {
```

Comment:
Can we add an additional test here, similar to this one, that tests support for `@TitleHeading` in a symbol extension? (Or is that already here and I'm just missing it maybe?) https://github.com/apple/swift-docc/pull/550/files is where this test was introduced.